esx_service/utils/auth.py

```diff
@@ -180,8 +178,7 @@ def get_tenant(vm_uuid):
             return error_msg, None, None
         if not tenant_uuid:
              vm_name = vmdk_utils.get_vm_name_by_uuid(vm_uuid)
-             err_code = error_code.ErrorCode.VM_NOT_BELONG_TO_TENANT
-             err_msg = error_code.error_code_to_message[err_code].format(vm_name)
+             err_msg = error_code_to_message[ErrorCode.VM_NOT_BELONG_TO_TENANT].format(vm_name)
```

Comment:
vm_name can be None?  If user has powered off the VM and removed it while the service is trying to execute this code?